README.md

```diff
@@ -39,6 +41,22 @@ from tree_sitter import Language, Parser
 PY_LANGUAGE = Language(tspython.language())
 ```
 
+#### Wasm support
+
+If you enable the `wasm` extra, then tree-sitter will be able to use wasmtime to load languages compiled to wasm and parse with them. Example:
```

Comment:
I don't know anything about comining tree-sitter and wasm (although I am familiar with both of them individually). To me this section doesn't really explain why you would compile a language to wasm. Perhaps that could be explained here.

If I guess correctly, the main reason is that wasm is cross-platform, so you don't need a separate language binary for each platform. Is this correct?

Reply:
That is correct.